compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -295,7 +295,7 @@ class ROCMTargetBackend final : public TargetBackend {
     fam.registerPass([&] { return targetMachine.getTargetIRAnalysis(); });
 
     llvm::PipelineTuningOptions pto;
-    pto.SLPVectorization = false;
+    pto.SLPVectorization = true;
```

Comment:
Can you make this togglable with a -test flag? Idk what the consensus on `llvm::cl::Hidden` is either as a hidden developer only flag.

Reply:
I decided to downgrade this change to an NFC with control on this option, so that downstream projects/ benchmark suites can test this out for a bit and we can make an informed choice based on that.